# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../xfsprogs/destdir.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./db/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./db/Makefile	2007-10-12 12:38:37.000000000 -0400
@@ -35,10 +35,10 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
-	$(INSTALL) -m 755 xfs_admin.sh $(PKG_BIN_DIR)/xfs_admin
-	$(INSTALL) -m 755 xfs_check.sh $(PKG_BIN_DIR)/xfs_check
-	$(INSTALL) -m 755 xfs_ncheck.sh $(PKG_BIN_DIR)/xfs_ncheck
-	$(INSTALL) -m 755 xfs_metadump.sh $(PKG_BIN_DIR)/xfs_metadump
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+	$(INSTALL) -m 755 xfs_admin.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_admin
+	$(INSTALL) -m 755 xfs_check.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_check
+	$(INSTALL) -m 755 xfs_ncheck.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_ncheck
+	$(INSTALL) -m 755 xfs_metadump.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_metadump
 install-dev:
--- ./io/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./io/Makefile	2007-10-12 12:40:17.000000000 -0400
@@ -64,9 +64,10 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 xfs_bmap.sh $(PKG_BIN_DIR)/xfs_bmap
-	$(LTINSTALL) -m 755 xfs_freeze.sh $(PKG_BIN_DIR)/xfs_freeze
-	$(LTINSTALL) -m 755 xfs_mkfile.sh $(PKG_BIN_DIR)/xfs_mkfile
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 xfs_bmap.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_bmap
+	$(LTINSTALL) -m 755 xfs_freeze.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_freeze
+	$(LTINSTALL) -m 755 xfs_mkfile.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_mkfile
+
 install-dev:
--- ./doc/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./doc/Makefile	2007-10-12 12:40:10.000000000 -0400
@@ -16,12 +16,12 @@
 	$(ZIP) --best -c < CHANGES > $@
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 PORTING CHANGES.gz CREDITS $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 PORTING CHANGES.gz CREDITS $(DESTDIR)$(PKG_DOC_DIR)
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -S CHANGES.gz $(PKG_DOC_DIR)/changelog.gz
+	$(INSTALL) -S CHANGES.gz $(DESTIDIR)$(PKG_DOC_DIR)/changelog.gz
 else
-	$(INSTALL) -m 644 COPYING $(PKG_DOC_DIR)
+	$(INSTALL) -m 644 COPYING $(DESTDIR)$(PKG_DOC_DIR)
 endif
 
 install-dev:
--- ./copy/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./copy/Makefile	2007-10-12 12:40:38.000000000 -0400
@@ -18,6 +18,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev:
--- ./fsck/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./fsck/Makefile	2007-10-12 12:40:56.000000000 -0400
@@ -12,6 +12,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_SBIN_DIR)
-	$(INSTALL) -m 755 xfs_fsck.sh $(PKG_SBIN_DIR)/fsck.xfs
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_SBIN_DIR)
+	$(INSTALL) -m 755 xfs_fsck.sh $(DESTDIR)$(PKG_SBIN_DIR)/fsck.xfs
+
 install-dev:
--- ./mkfs/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./mkfs/Makefile	2007-10-12 12:41:20.000000000 -0400
@@ -26,6 +26,7 @@
 	$(LTLINK) $@.c -o $@ $(CFLAGS) $(LDFLAGS) $(LIBDISK) $(PLDLIBS)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_SBIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_SBIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_SBIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_SBIN_DIR)
+
 install-dev:
--- ./rtcp/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./rtcp/Makefile	2007-10-12 12:41:37.000000000 -0400
@@ -14,6 +14,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev:
--- ./Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./Makefile	2007-10-12 12:42:07.000000000 -0400
@@ -53,8 +53,8 @@
 
 install: default
 	$(SUBDIRS_MAKERULE)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 README $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 README $(DESTDIR)$(PKG_DOC_DIR)
 
 install-dev: default
 	$(SUBDIRS_MAKERULE)
--- ./quota/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./quota/Makefile	2007-10-12 12:42:22.000000000 -0400
@@ -33,6 +33,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev:
--- ./debian/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./debian/Makefile	2007-10-12 12:43:26.000000000 -0400
@@ -15,20 +15,20 @@
 
 install: default
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 changelog $(PKG_DOC_DIR)/changelog.Debian
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 changelog $(DESTDIR)$(PKG_DOC_DIR)/changelog.Debian
 endif
 
 install-dev: default
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 755 -d $(DEV_DOC_DIR)
-	$(INSTALL) -m 644 copyright $(DEV_DOC_DIR)
-	$(INSTALL) -m 644 changelog $(DEV_DOC_DIR)/changelog.Debian
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(DEV_DOC_DIR)
+	$(INSTALL) -m 644 copyright $(DESTDIR)$(DEV_DOC_DIR)
+	$(INSTALL) -m 644 changelog $(DESTDIR)$(DEV_DOC_DIR)/changelog.Debian
 endif
 
 install-d-i: default
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -m 755 -d $(PKG_SBIN_DIR)
-	$(INSTALL) -m 755 $(BOOT_MKFS_BIN) $(PKG_SBIN_DIR)/mkfs.xfs
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_SBIN_DIR)
+	$(INSTALL) -m 755 $(BOOT_MKFS_BIN) $(DESTDIR)$(PKG_SBIN_DIR)/mkfs.xfs
 endif
--- ./growfs/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./growfs/Makefile	2007-10-12 12:43:47.000000000 -0400
@@ -19,7 +19,8 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
-	$(INSTALL) -m 755 xfs_info.sh $(PKG_BIN_DIR)/xfs_info
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+	$(INSTALL) -m 755 xfs_info.sh $(DESTDIR)$(PKG_BIN_DIR)/xfs_info
+
 install-dev:
--- ./repair/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./repair/Makefile	2007-10-12 12:45:37.000000000 -0400
@@ -45,6 +45,7 @@
 #CFLAGS += ...
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_SBIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_SBIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_SBIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_SBIN_DIR)
+
 install-dev:
--- ./logprint/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./logprint/Makefile	2007-10-12 12:45:53.000000000 -0400
@@ -21,6 +21,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev:
--- ./mdrestore/Makefile.orig	2007-10-12 12:37:26.000000000 -0400
+++ ./mdrestore/Makefile	2007-10-12 12:46:12.000000000 -0400
@@ -17,6 +17,7 @@
 include $(BUILDRULES)
 
 install:
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev:
--- ./include/Makefile.orig	2007-10-12 13:03:00.000000000 -0400
+++ ./include/Makefile	2007-10-12 13:04:40.000000000 -0400
@@ -46,8 +46,8 @@
 include $(BUILDRULES)
 
 install-dev: default
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)
-	$(INSTALL) -m 644 $(HFILES) $(PKG_INC_DIR)
-	$(INSTALL) -m 644 platform_defs.h $(PKG_INC_DIR)
-	$(INSTALL) -m 755 -d $(DK_INC_DIR)
-	$(INSTALL) -m 644 $(DKHFILES) $(DK_INC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)
+	$(INSTALL) -m 644 $(HFILES) $(DESTDIR)$(PKG_INC_DIR)
+	$(INSTALL) -m 644 platform_defs.h $(DESTDIR)$(PKG_INC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(DK_INC_DIR)
+	$(INSTALL) -m 644 $(DKHFILES) $(DESTDIR)$(DK_INC_DIR)
--- ./include/buildmacros.orig	2007-10-12 13:09:39.000000000 -0400
+++ ./include/buildmacros	2007-10-12 13:14:34.000000000 -0400
@@ -55,9 +55,9 @@
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -m 644 -T so_dot_version $(LIBNAME).lai $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_dot_current $(LIBNAME).lai $(PKG_LIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -m 644 -T so_dot_version $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_dot_current $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR)
 endif
 
 # Libtool thinks the static and shared libs should be in the same dir, so
@@ -69,22 +69,22 @@
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB_DEV = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 $(LIBNAME).lai $(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_base $(LIBNAME).lai $(PKG_LIB_DIR); \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(PKG_LIB_DIR)/$(LIBNAME).a; \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(PKG_LIB_DIR)/$(LIBNAME).la; \
-	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(PKG_DEVLIB_DIR)/$(LIBNAME).so
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_base $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).a; \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).la; \
+	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).so
 else
 INSTALL_LTLIB_DEV = $(INSTALL_LTLIB_STATIC)
 endif
 
 INSTALL_LTLIB_STATIC = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)
 
 INSTALL_MAN = \
 	@for d in $(MAN_PAGES); do \
@@ -96,7 +96,7 @@
 				-e 's/\\\f[0-9]//g' -e 's/  / /g;q'`; \
 		do \
 			[ -z "$$m" -o "$$m" = "\\" ] && continue; \
-			t=$(MAN_DEST)/$$m.$(MAN_SECTION); \
+			t=$(DESTDIR)$(MAN_DEST)/$$m.$(MAN_SECTION); \
 			if $$first; then \
 				if $(HAVE_ZIPPED_MANPAGES); then \
 					$(ZIP) -9 -c $$d > $$d.gz; _sfx=.gz; \
@@ -116,7 +116,7 @@
 INSTALL_LINGUAS = \
 	@for l in $(LINGUAS) ""; do \
 		if test -f "$$l.mo" ; then \
-			ldir=$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
+			ldir=$(DESTDIR)$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
 			$(INSTALL) -m 755 -d $$ldir; \
 			$(INSTALL) -m 644 $$l.mo $$ldir/$(PKG_NAME).mo; \
 		fi; \
